gramps/gui/plug/_windows.py

```diff
@@ -1072,16 +1072,16 @@ def __init__(self, dbstate, uistate, options_class, name, callback=None):
 # UpdateAddons
 #
 #-------------------------------------------------------------------------
-class UpdateAddons:
+class UpdateAddons(ManagedWindow):
 
-    def __init__(self, addon_update_list, parent_window):
+    def __init__(self, uistate, track, addon_update_list):
         self.title = _('Available Gramps Updates for Addons')
 
+        ManagedWindow.__init__(self, uistate, track, self, modal=True)
         glade = Glade("updateaddons.glade")
-        self.window = glade.toplevel
+        self.set_window(glade.toplevel, None, None)
         self.window.set_title(self.title)
-        self.window.set_size_request(750, 400)
-        self.window.set_transient_for(parent_window)
+        self.setup_configs("interface.addonscheck", 750, 400)
```

Comment:
Are you suggesting that we actually rename the _class_?  Or that I should rename the config file "addonscheck" to "updateaddons"?  I did not think that there had to be a direct correspondence between the config interface element name and the class (at least there doesn't seem to be a direct link in a number of cases).